### PR TITLE
T3T1 status animation

### PR DESCRIPTION
**Pull request by TychoVrahe**
adds status animation to mercury UI

### CODE REVIEW

**Review by obrusvit — APPROVED** [review]
utACK

File: core/embed/rust/src/ui/model_mercury/component/status_screen.rs [hunk 1]
```diff
@@ -1,13 +1,76 @@
...
+
+    pub fn get_circle_scale(&self, t: f32) -> i16 {
+        let circle_scale = pareen::constant(0.0).seq_ease_out(
+            0.2,
+            easer::functions::Cubic,
+            0.4,
+            pareen::constant(1.0),
+        );
+
+        i16::lerp(170 / 2, 80 / 2, circle_scale.eval(t))
```
Comment (obrusvit) [inline core/embed/rust/src/ui/model_mercury/component/status_screen.rs:62]: well, would name these constants at least `170`, `80`. And not clear why divided by two. 
Not blocking the merge over this.

Comment (TychoVrahe) [reply core/embed/rust/src/ui/model_mercury/component/status_screen.rs]: https://github.com/trezor/trezor-firmware/pull/3807/commits/a65f355680f7a7404520b104253d5546e638ec13

**Comment by STew790** [comment]
**QA OK**
Animation is OK

**Info:**
 - OS: Linux x86_64
 - Screen: 1920x1080
 - Device: Trezor T3T1 2.7.1 regular (revision 924492ce1b40656762296516734ee91201fb2a22)
 - Transport: BridgeTransport 2.0.33

**Pull request merged**